lib/workers/repository/process/fetch.ts

```diff
@@ -15,6 +16,9 @@ async function fetchDepUpdates(
 ): Promise<PackageDependency> {
   let dep = clone(indep);
   dep.updates = [];
+  if (!is.nonEmptyString(dep.depName)) {
```

Comment:
```suggestion
  if (!is.nonEmptyString(dep.depName?.trim)) {
```

Please add tests for `null`, `undefined` and `' '`, `''` to make sure this is behaving as expected :)